glade-ad-single-listener.html

```diff
@@ -0,0 +1,103 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="/bower_components/polymer/polymer.html">
+<dom-module id="glade-ad">
+    <template>
+        <style>
+         :host {
+            display: block;
+            width: 100%;
+            height: 100%;
+        }
+        </style>
+        <div class="ad_slot_placeholder"></div>
+        </div>
+    </template>
+    <script>
+        (function() {
+            var ad_slot;
+            var gladeReady = !!(window.glade && window.glade.run); // Maybe Glade has already loaded?
+            var elementsWaitingForGlade = [];
+            if (!gladeReady) {
+              window.addEventListener('gladeReady', function listener() {
+                  window.removeEventListener('gladeReady', listener);
+                  gladeReady = true;
+                  for (var i = 0; i < elementsWaitingForGlade.length; i++) {
+                      elementsWaitingForGlade[i]._load();
+                  }
+              });
+            }
+
+            Polymer({
+                is: 'glade-ad',
+                properties: {
+                    adSlotId: {
+                        type: String,
+                    },
+                    dataAdUnitPath: {
+                        type: String
+                    },
+                    width: {
+                        type: String
+                    },
+                    height: {
+                        type: String
+                    },
+                    dataClickUrl: {
+                        type: String
+                    },
+                    dataPageUrl: {
+                        type: String
+                    },
+                    dataJson: {
+                        type: String
+                    }
+                },
+
+                observers: [
+                    'propertiesChanged(adSlotId, dataAdUnitPath, width, height)'
```

Comment:
`propertiesChanged` no longer exists, so I think you can remove this whole `observers` list.

(tangent) If I remember correctly, all of the attributes will have been set to their respective properties by the time `ready` is called. Given that a `<glade-ad>` always waits until `ready` before it attempts to run `_load` (either running sync or inserting itself into the queue to be called later), all the properties should be available in time.

Reply:
Got it; removing the observers list.